.eslintrc.json

```diff
@@ -31,26 +31,12 @@
     "no-inner-declarations": "off",
     "valid-jsdoc": "off",
     "require-jsdoc": "off",
-    "quotes": ["error", "backtick"],
```

Comment:
Prettier is taking care of the quotes.. and do we really want to use back-ticks for all quotes?!

Reply:
Prettier doesn't convert backticks to quotes. https://github.com/prettier/prettier/issues/2996#issuecomment-335137564
Backticks are used across the entire codebase even when there is nothing to interpolate, also in the documentation. This was a deliberate decision: see https://github.com/prettier/prettier/issues/54 for discussion.

Reply:
documentation part I can agree on - we shouldn't "enforce" back-ticks there, but this shouldn't mean we need to change code style in non-docs part of the repo, right?

Reply:
I know stuff like this can get people polarized, but can we not "sneak in" change like that with this PR? Can we just setup hooks to format things here and have separate PR or issue to discuss code style changes?

Reply:
Kept that option.. just sorted the rules :)